### DISCUSSION TITLE
Cannot assign repository's team under Reviewers in pull request

### DISCUSSION THREAD

**Discussion by karolkarolka** [community/community#45231] [Pull Requests]
### Select Topic Area

Bug

### Body

As stays in the title, I cannot assign my team/group or even its child team as a reviewer in PRs. I can easily mention the team within the comment/description but after typing the team's name under Reviewers section in PR, the text "Nothing to show" appears. Is this feature restricted to private repos on paid GitHub plans?

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review
<img width="729" alt="Zrzut ekranu 2023-01-24 o 17 29 15" src="https://user-images.githubusercontent.com/13152501/214350841-6fa1f94d-4500-44ff-9e4a-e18fbd037514.png">

**Comment by gsanchezgavier** [comment]
I'm experiencing the same issue. After some weeks of inactivity, I cannot assign the team I belong as a reviewer of a PR . The team is set to visible and other members of the team have not this issue.
